setup.py

```diff
@@ -93,7 +93,7 @@
             'projects = cfme.configure.access_control:ProjectCollection',
             'candus = cfme.configure.configuration.region_settings:CANDUCollection',
             'groups = cfme.configure.access_control:GroupCollection',
-            'nodes = cfme.containers.node:NodeCollection',
+            'container_nodes = cfme.containers.node:NodeCollection',
```

Comment:
I updated nodes to container_nodes to keep consistency, tests have been updated accordingly.